toolkit/scripts/toolchain.mk

```diff
@@ -33,24 +35,29 @@ TOOLCHAIN_MANIFEST ?= $(TOOLCHAIN_MANIFESTS_DIR)/toolchain_$(build_arch).txt
 # the exact path of the required rpm
 # Outputs: $(toolchain_rpms_dir)/<arch>/<name>.<arch>.rpm
 sed_regex_full_path = 's`(.*\.([^\.]+)\.rpm)`$(toolchain_rpms_dir)/\2/\1`p'
+sed_regex_full_path_rehydrated = 's`(.*\.([^\.]+)\.rpm)`$(toolchain_from_repos)/\1`p'
 toolchain_rpms := $(shell sed -nr $(sed_regex_full_path) < $(TOOLCHAIN_MANIFEST))
 toolchain_rpms_buildarch := $(shell grep $(build_arch) $(TOOLCHAIN_MANIFEST))
 toolchain_rpms_noarch := $(shell grep noarch $(TOOLCHAIN_MANIFEST))
+toolchain_rpms_rehydrated := $(shell sed -nr $(sed_regex_full_path_rehydrated) < $(TOOLCHAIN_MANIFEST))
 
 $(call create_folder,$(toolchain_build_dir))
 $(call create_folder,$(toolchain_downloads_logs_dir))
+$(call create_folder,$(toolchain_from_repos))
 $(call create_folder,$(populated_toolchain_chroot))
 
 .PHONY: raw-toolchain toolchain clean-toolchain check-manifests check-aarch64-manifests check-x86_64-manifests
 raw-toolchain: $(raw_toolchain)
 toolchain: $(toolchain_rpms)
+partially-rehydrate-toolchain-from-repo: $(toolchain_rpms_rehydrated)
```

Comment:
Where is "partially-rehydrate-toolchain-from-repo" referenced?

Reply:
It's not, it's a leftover from testing- made things easier to have a target that covered all of the rehydrated RPMs. I'll go ahead and remove this, since there's no usage scenario for this.